reference/toolstacks/php/drupal/redis-drupal-8.md

```diff
@@ -0,0 +1,163 @@
+Platform.sh's environment variables configuration, so using `settings.php` directly
+is the recommended approach.
+
+Place the following at the end of `settings.php`. Note the inline comments, as you may wish to customize
+it further.  Also review the `README.txt` file that comes with the redis module,
+as it has a great deal more information on possible configuration options. For instance,
+you may wish to not use Redis for the persistent lock if you have a custom module 
+that needs locks to persist for more than a few seconds.
+
+The example below is intended as a "most common case".
+
+```php
+// Set redis configuration.
+if (!empty($_ENV['PLATFORM_RELATIONSHIPS']) && extension_loaded('redis')) {
+  $relationships = json_decode(base64_decode($_ENV['PLATFORM_RELATIONSHIPS']), TRUE);
+
+  if (!empty($relationships['redis'][0])) {
+    $redis = $relationships['redis'][0];
+
+    $settings['cache']['default'] = 'cache.backend.redis';
+    $settings['redis.connection']['host'] = $redis['host'];
+    $settings['redis.connection']['port'] = $redis['port'];
+
+    // Enable the redis server overrides, e.g. cache tags checksum.
+    // See that file, you might want to copy and customize it, it is meant as an example
+    // and will change as more backends are offered by the redis module.
+    $settings['container_yamls'][] = 'modules/contrib/redis/example.services.yml';
+
+    // Manually add the classloader path, this is required for the container cache bin definition below
+    // and allows to use it without the redis module being enabled.
+    $class_loader->addPsr4('Drupal\\redis\\', 'modules/contrib/redis/src');
+
+    // Use redis for container cache.
+    $settings['bootstrap_container_definition'] = [
+      'parameters' => [],
+      'services' => [
+        'redis.factory' => [
+          'class' => 'Drupal\redis\ClientFactory',
+        ],
+        'cache.backend.redis' => [
+          'class' => 'Drupal\redis\Cache\CacheBackendFactory',
+          'arguments' => ['@redis.factory', '@cache_tags_provider.container'],
+        ],
+        'cache.container' => [
+          'class' => '\Drupal\redis\Cache\PhpRedis',
+          'factory' => ['@cache.backend.redis', 'get'],
+          'arguments' => ['container'],
+        ],
+        'cache_tags_provider.container' => [
+          'class' => 'Drupal\redis\Cache\RedisCacheTagsChecksum',
+          'arguments' => ['@redis.factory'],
+        ],
+      ],
+    ];
+
+    // Explicitly set the fast backend for bootstrap, default, discover and config,
+    // otherwise this gets lost when redis is enabled.
+    $settings['cache']['bins']['bootstrap'] = 'cache.backend.chainedfast';
+    $settings['cache']['bins']['discovery'] = 'cache.backend.chainedfast';
+    $settings['cache']['bins']['config'] = 'cache.backend.chainedfast';
+    // Default does not use the fast chained backend by default, but core only uses it for small caches
```

Comment:
Well that got committed fast. :smile: 

@Berdir So if I follow you correctly, as of 8.2 this entire block from line 127 to 137 can go away in 8.2?  It's all just to work around that bug?  If so, I'll definitely switch up the description and mark it to remove later.